src/bloom.cpp

```diff
@@ -34,12 +34,12 @@ CBloomFilter::CBloomFilter(unsigned int nElements, double nFPRate, unsigned int
      * Again, we ignore filter parameters which will create a bloom filter with more hash functions than the protocol limits
```

Comment:
let's also fix the nElements == 0 DIV0 error in this by setting nElements to 1 if its zero.